railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -272,6 +273,9 @@ class AppGenerator < AppBase
       class_option :api, type: :boolean,
                          desc: "Preconfigure smaller stack for API only apps"
 
+      class_option :interactive, type: :boolean, default: false, aliases: "--i",
```

Comment:
Shouldn't it be conventionally either `-i` or `--interactive`? Like one dash for a single letter and two dashes for the whole word

Reply:
Good catch!